atlas/templates/cmd/server/main.go.gotmpl

```diff
@@ -162,6 +165,7 @@ func forwardResponseOption(ctx context.Context, w http.ResponseWriter, resp prot
 	return nil
 }
 {{- end }}
+
```

Comment:
Correct me if I am wrong but this line redundant due to upper {{- end}} statement will be translated into empty line